mesa/space.py

```diff
@@ -504,11 +504,10 @@ def position_agent(
     ) -> None:
         """Position an agent on the grid.
         This is used when first placing agents! Setting either x or y to "random"
-        gives the same behavior as 'move_to_empty()'.
-        Use 'swap_pos()' to swap agents positions.
-        If x or y are positive, they are used, but if "random",
-        we get a random position.
+        gives the same behavior as 'move_to_empty()' to get a random position.
+        If x or y are positive, they are used.
         Ensure this random position is not occupied (in Grid).
```

Comment:
I think this line can be removed.
> Ensure this random position is not occupied (in Grid).

Reply:
Sure!
[24e01b0](https://github.com/projectmesa/mesa/pull/1457/commits/24e01b09cfaeb5a625b0b9a0fc3be23f29e0633f) - Update docstring to delete unnecessary sentence